### PR TITLE
#267 Fix nrt

### PR DESCRIPTION
**Pull request by cossartque**
**Checklist before requesting a review**

*use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes*

- [ ] unit tests and non regression tests were added (update of algorithms)
- [ ] main documentation was updated (update of input/output file, update of algorithms)
- [ ] if this PR modifies a dictionary: the corresponding french dictionary was updated
- [ ] if this commit targets a release branch: the corresponding milestone was added in the ticket and in this PR

### CODE REVIEW

File: nrt/data/IEEE14/MC_file_div/reference/aggregatedResults.xml [hunk 1]
```diff
@@ -11,15 +11,15 @@
     <scenarioResults id="DisconnectGroup" status="CONVERGENCE"/>
   </loadIncreaseResults>
   <loadIncreaseResults loadLevel="7" status="CONVERGENCE">
-    <scenarioResults id="DisconnectLine" status="CONVERGENCE"/>
```
Comment (cossartque) [inline nrt/data/IEEE14/MC_file_div/reference/aggregatedResults.xml:14]: This nrt might need a double check as the results are different now and I don't really know what might cause it

Comment (rosiereflo) [reply nrt/data/IEEE14/MC_file_div/reference/aggregatedResults.xml:14]: It is a known issue, @marcochiaramello is working on it

Comment (cossartque) [reply nrt/data/IEEE14/MC_file_div/reference/aggregatedResults.xml:14]: Ok, so we close this MR then ?

Comment (rosiereflo) [reply nrt/data/IEEE14/MC_file_div/reference/aggregatedResults.xml:14]: I think we can keep the modifications in the criteria and revert this one

**Review by rosiereflo — APPROVED** [review]

**Pull request merged**